client/displayservers/Wayland/input.c

```diff
@@ -413,3 +413,17 @@ void waylandRealignPointer(void)
   if (!wlWm.warpSupport)
     app_resyncMouseBasic();
 }
+
+void waylandGuestPointerUpdated(double x, double y, int localX, int localY)
+{
+  if (!wlWm.warpSupport || !wlWm.pointerInSurface || app_isCaptureMode())
+    return;
+
+  static uint64_t last_warp = 0;
+  uint64_t now = microtime();
+  if (now - last_warp < 10000)
+    return;
+  last_warp = now;
```

Comment:
Not sure if this throttling is necessary. There appears to be no added CPU usage on sway from warping as far as I can tell from watching `htop`.

Reply:
I needed to include this as rounding errors could cause multiple leave/enter events in X11 due to excessive warp commands in the X queue. It may not apply at all to Wayland.

Reply:
It appears to be unnecessary. Removed.